src/domain/consents/{ID}/generateChallenge.ts

```diff
@@ -0,0 +1,118 @@
+/* istanbul ignore file */
```

Comment:
we don't need {ID} in domain's  folder structure -> it is only required for handlers

Reply:
I'm confused so then should the structure just be 
domain/consents/generateChallenge?